base/shell/cmd/batch.c

```diff
@@ -402,6 +403,11 @@ BOOL BatchGetString (LPTSTR lpBuffer, INT nBufferLength)
     INT len = 0;
 #ifdef _UNICODE
     lpString = cmd_alloc(nBufferLength);
+    if (!lpString)
+    {
+        WARN("DEBUG: Cannot allocate memory for lpString\n");
+        return 1;
```

Comment:
Why is error_out_of_memory not used here?
And might wanna return TRUE or FALSE, and most likely FALSE here.